iquip/apps/dataviewer.py

```diff
@@ -110,6 +110,18 @@ def nearestDataPoint(
         """
         return None
 
+    def highlight(self, index: Optional[Tuple[int, ...]]):
+        """Highlights the data point at the given index.
+        
+        A vertical and a horizontal lines appear at the data point.
+        If there was already a highlight lines, they move to the new point, i.e.,
+          there are at most one highlighted data point at once.
```

Comment:
Umm.. I think this explanation should be moved into `CurvePlotViewer` because we may want to implement this differently according to subclasses of `NDArrayViewer` and now you overrode it in only `CurvePlotViewer`.

Reply:
It makes sense, it does not have to be two lines for the other viewers. I will move that docstring.